Automattic/wp-calypso
# Tests- Attempt to fix VR build report and broken tests

#### Changes proposed in this Pull Request

* This PR tries to fix the build tab to show the embedded report. 
* It also fixes some tests that were broken by changes to the theme.

The build report tab doesn't show on the build below. I think it may need to be merged in for it to work. It won't harm anything if it doesn't work.

#### Testing instructions

* Make sure tests [pass in TC](https://teamcity.a8c.com/viewType.html?buildTypeId=calypso_VisualRegressionTests&branch_calypso_WPComTests=fix%2Fvr-build-tab&tab=buildTypeStatusDiv)
